# Figure 2 c/d from the Dorrestein Bile Acid Review for Nature Gastroenterology and Hepatology

### Figure 2 Legend: 
Distribution of bile acids and related transcripts/proteins. Distribution of observed bile acids as documented with the mass spectrometry reanalysis of data user interface in a) human and b) mouse. For transcript locations, all matches discussed are to “bile” in the c) [Human Protein Atlas](https://www.proteinatlas.org/) and d) [mouse gene expression database](http://www.informatics.jax.org/expression.shtml). Black circles represent the number of bile acid-associated genes or gene products observed in respective body locations. Overlaid is the information about the bile acid receptor proteins, FXR, SXR (humans), PXR (mouse), BAAT, and TGR5. All panels are made using Biorender.com. It should be noted that inferences of differential distribution should not be inferred between organisms from this figure but rather as a documentation where transcripts, proteins, and bile acids have been detected. It is also an incomplete picture as many organs or biofluids have not been studied by transcriptomics, proteomics or metabolomics, let alone under all conditions that may alter the detection but gives an overview just how widely the bile acid metabolic and gene network is distributed throughout the body. BSH = bile salt hydrolase, BAAT = Bile acid-CoA:amino acid N-acyltransferase.

### Figure 2 c/d Discussion:

There are bile acid-associated proteins, transporters, and receptors in all tissues that have been analyzed by proteomics, consistent with a systemic role of bile acids as opposed to those proteins that serve a role in the enterohepatic circulatory system (Figure 2c, d). Some of the bile acid receptors like FXR, PXR, and TGR5 that are extensively studied in the liver and intestine, can also be detected in distant tissues such as the brain, adipose tissue, gallbladder and the breast (Figure 2c, d). 



# Permanent Link
#### DOI:

 
